mods/d2k/rules/aircraft.yaml

```diff
@@ -1,46 +1,98 @@
-^CARRYALL:
+carryall.scripted:
 	Inherits: ^Helicopter
 	Valued:
 		Cost: 1200
 	Tooltip:
 		Name: Carryall
 		Description: Large winged, planet-bound ship\n  Automatically lifts harvesters.
-	Buildable:
-		Queue: Aircraft
-		BuildPaletteOrder: 120
 	Health:
 		HP: 250
 	Armor:
 		Type: Light
 	Helicopter:
-		CruiseAltitude: 2100
+		CruiseAltitude: 2048
 		InitialFacing: 0
 		ROT: 4
 		Speed: 160
 		LandableTerrainTypes: Sand, Rock, Transition, Spice, Dune
-		RepairBuildings: repaira,repairo,repairh
-		RearmBuildings: starporta,starporto,starporth
+		RepairBuildings: repair
+		RearmBuildings:
 		Repulsable: False
-		LandAltitude: 100
+		LandAltitude: 512
 		LandWhenIdle: False
+	RenderUnit:
+		Image: carryall
 	WithShadow:
 	LeavesHusk:
-		HuskActor: CARRYALL.Husk
+		HuskActor: carryall.husk
 	-Selectable:
+
+carryall:
+	Inherits: carryall.scripted
```

Comment:
Not really - this adds an additional trait.  The yaml merger has bugs that choke on inherited removals in some cases (like this).